Core.Auditing/IfxAuditLogger.cs

```diff
@@ -116,5 +118,26 @@ public static string FetchIPAddress()
 
             return ipAddress;
         }
```

Comment:
nitpick: empty line after method

Reply:
Still no empty line?

Reply:
Do we need an empty line because I don't see one??